Formula/htmldoc.rb

```diff
@@ -16,14 +17,19 @@ class Htmldoc < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "libpng"
 
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "gnutls"
+  end
+
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-ssl",
-                          "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+    system "./configure", *std_configure_args,
+                          "--mandir=#{man}",
+                          "--without-gui"
```

Comment:
SSL was disabled in https://github.com/Homebrew/homebrew-core/commit/d64467f7fc1f186e2e1abf0f4d2d65cdeede343d. It looks like upstream has been using Security/CoreFoundation frameworks on macOS even in 1.8.29 (https://github.com/michaelrsweet/htmldoc/blob/release-1.8.29/configure.ac#L159), so not sure what exact issue was back then. Maybe particular OS X / macOS version support for framework?

GUI has been indirectly disabled by missing `fltk` dependency, but just manually set to avoid building if `fltk` is available in build environment.